### DISCUSSION TITLE
How do I use this in a console service?

### DISCUSSION THREAD

**Discussion by phillijw** [PrometheusClientNet/Prometheus.Client#196] [Q&A] [answered]
The instructions don't seem to provide any suggestions for how to host the collection endpoint when not using aspnet, unless I'm misunderstanding the README.

**Comment by phillijw** [comment]
Ah, is this it?

https://github.com/prom-client-net/prom-client-metricserver

**Comment by phnx47** [comment, marked as answer]
@phillijw Yes, you can use https://github.com/prom-client-net/prom-client-metricserver, examples here: https://github.com/prom-client-net/prom-examples/tree/master/MetricServer
